webapi/Controllers/DocumentImportController.cs

```diff
@@ -276,16 +292,41 @@ private async Task ValidateDocumentImportFormAsync(DocumentImportForm documentIm
                 case SupportedFileType.Jpg:
                 case SupportedFileType.Png:
                 case SupportedFileType.Tiff:
-                {
-                    if (this._ocrSupportOptions.Type != OcrSupportOptions.OcrSupportType.None)
+                    if (this._ocrSupportOptions.Type == OcrSupportOptions.OcrSupportType.None)
```

Comment:
Is this change correct?

Reply:
No, I keep changing it back and forth for testing, should be != now